local-cluster/tests/local_cluster.rs

```diff
@@ -1536,20 +1538,32 @@ fn test_wait_for_max_stake() {
     solana_logger::setup_with_default(RUST_LOG_FILTER);
     let validator_config = ValidatorConfig::default_for_test();
     let slots_per_epoch = MINIMUM_SLOTS_PER_EPOCH;
+    // Set this large enough to allow for skipped slots but still be able to
+    // make a root and derive the new leader schedule in time.
+    let stakers_slot_offset = slots_per_epoch.saturating_mul(MAX_LEADER_SCHEDULE_EPOCH_OFFSET);
```

Comment:
This piece definitely looks to give us more "wiggle room" in regards to the cluster hitting the scenario you described of not being able to vote and getting stuck